### ISSUE TITLE
ZenFS contexts

### ISSUE THREAD

**Issue by james-pre** [anbraten/nano-web-ide#1]
Hi, I'm the ZenFS maintainer. I noticed the TODO comment:

https://github.com/anbraten/nano-web-ide/blob/90776db1d2d5e2551da7be24055db95e544804d4/packages/core/src/fs.ts#L3

Just letting you know, contexts were released in `@zenfs/core` 1.4.0.

**Comment by anbraten** [comment]
@james-pre Awesome and thanks for stopping by 😊 Can I even change the user to throw permission errors with sth like `bindContext('/home/anbraten', credentials)` now? A volume like from `memfs` would just be created by using some random base path per `bindContext`, right?

**Comment by james-pre** [comment]
Yes, you can now change credentials for a context, which you could use as the basis for a process. For "volumes" you could do something like what you're describing.
